CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to
 
 ### Added
 
+- Publish Storybook through CircleCI
```

Comment:
This is something related to the internal behavior of the project; this is not needed into the changelog.